metricbeat/module/vsphere/host/data.go

```diff
@@ -75,23 +75,23 @@ func mapPerfMetricToEvent(event mapstr.M, perfMetricMap map[string]interface{})
 		event.Put("disk.latency.total.ms", val)
 	}
 	if val, exist := perfMetricMap["disk.usage.average"]; exist {
-		event.Put("disk.total.bytes", val.(int64)*1000)
+		event.Put("disk.total.bytes", val.(int64)*1024)
 	}
 	if val, exist := perfMetricMap["disk.read.average"]; exist {
-		event.Put("disk.read.bytes", val.(int64)*1000)
+		event.Put("disk.read.bytes", val.(int64)*1024)
 	}
 	if val, exist := perfMetricMap["disk.write.average"]; exist {
-		event.Put("disk.write.bytes", val.(int64)*1000)
+		event.Put("disk.write.bytes", val.(int64)*1024)
 	}
 
 	if val, exist := perfMetricMap["net.transmitted.average"]; exist {
-		event.Put("network.bandwidth.transmitted.bytes", val.(int64)*1000)
+		event.Put("network.bandwidth.transmitted.bytes", val.(int64)*1024)
 	}
 	if val, exist := perfMetricMap["net.received.average"]; exist {
-		event.Put("network.bandwidth.received.bytes", val.(int64)*1000)
+		event.Put("network.bandwidth.received.bytes", val.(int64)*1024)
 	}
 	if val, exist := perfMetricMap["net.usage.average"]; exist {
-		event.Put("network.bandwidth.total.bytes", val.(int64)*1000)
+		event.Put("network.bandwidth.total.bytes", val.(int64)*1024)
```

Comment:
Use a multiplier to have consistency across code?
https://github.com/elastic/beats/pull/40485/files#diff-9cc181afc4df8bc033203a07032b99915b2291d0d7db71cce27db73f75a7872bR25

Reply:
Got it, will do that.